src/mugs.js

```diff
@@ -1508,6 +1546,7 @@ define([
             // Nestable Field List not supported in CommCare before v2.16
             group_in_field_list = opts.features.group_in_field_list,
             richText = opts.features.rich_text;
+        ENABLE_IMAGE_RESIZER = opts.features.image_resize;
```

Comment:
Is it feasible to set a `resizeEnabled` flag on the Image mug instead of using a global flag?